microsite-next/src/pages/demos/index.tsx

```diff
@@ -0,0 +1,3 @@
+import { Demos } from './_demos';
+
+export default Demos;
```

Comment:
We could, but I personally think the folder structure is easier to read if we have the file with the name of the component, the only reason we need this index file is because of Docusaurus routing, so can't drop the index file

Reply:
I like the folder structure but I was thinking we could keep the component directly in the index file instead.

Reply:
Sure, will change